docs/source/index.rst

```diff
@@ -3,31 +3,62 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to MetaSynth's documentation!
-=====================================
 
-MetaSynth is a package for generating statistical metadata from polars (or pandas) dataframes and creating synthetic data from this metadata.
 
-It can be used when private datasets cannot be shared directly. By creating a synthetic dataset,
-the data is safeguarded against the risk of disclosure. On the scale of disclosure/utilitly as defined
-by the `Office for National Statistics <https://www.ons.gov.uk/methodology/methodologicalpublications/generalmethodology/onsworkingpaperseries/onsmethodologyworkingpaperseriesnumber16syntheticdatapilot>`_,
-MetaSynth resides in the *Synthetic-augmented plausible* category. This means that individual columns
-should bear statistical/univariate resemblence to the original dataset, but the relationships between
-the columns in the dataset is lost.
+.. image:: /images/logos/blue.svg
+   :alt: MetaSynth_Logo
 
-To create a metadata file for your dataset, you need to create a polars dataframe. For a simple example
-see the :doc:`quick_start`.
+MetaSynth Documentation
+=======================
+Welcome to the MetaSynth documentation. MetaSynth is a Python package that facilitates the generation of synthetic data and metadata, enabling users to create surrogate datasets that mimic the structure and characteristics of their original data while preserving privacy.
```

Comment:
It would be nice to have a link to the GitHub page somewhere on this page.

Reply:
Latest version embeds a link in the first 'MetaSynth', I will also add an explicit link in the 'about' page.